OpenRVS/classes/OpenClientBeaconReceiver.uc

```diff
@@ -58,40 +57,37 @@ event ReceivedText(IpAddr Addr,string Text)
 			sModName = ParseOption(szThirdWord,ModNameMarker);
 			//1.3 - sModName string added to function in MP menu
 			Widget.ReceiveServerInfo(IpAddrToString(Addr),sNumP,sMaxP,sGMode,sMapName,sSvrName,sModName);//send received info back to server list
-			//debug:
-//			log("** Server " $ sSvrName $ " at " $ IpAddrToString(Addr) $ " is playing map " $ sMapName $ " in game mode type " $ sGMode $ ". Players: " $ sNumP $ "/" $ sMaxP);//debug
+			class'OpenLogger'.static.DebugLog("Server " $ sSvrName $ " at " $ IpAddrToString(Addr) $ " is playing map " $ sMapName $ " in game mode type " $ sGMode $ ". Players: " $ sNumP $ "/" $ sMaxP);//debug
 		}
 	}
 }
 
 //overridden from parent
 //need to strip out the final space from result
 //also need to add the removed symbol back into result
-function bool GrabOption(out string Options,out string Result)//�I1 OBSOLETESUPERSTARS.COM �F1 RGM
+function bool GrabOption(out string Options,out string Result)//¶I1 OBSOLETESUPERSTARS.COM ¶F1 RGM
 {
-	if ( Left(Options,1) == "�" )
+	if ( Left(Options,1) == "¶" )

```

Comment:
still not entirely sure why this line is showing as changed. seems like the same pilcrow sign and no whitespace changes.